metals/src/main/scala/scala/meta/internal/metals/StacktraceAnalyzer.scala

```diff
@@ -0,0 +1,227 @@
+package scala.meta.internal.metals
+
+import java.io.FileWriter
+import java.net.URLEncoder
+import java.util.Collections.singletonList
+
+import scala.io.Source
+import scala.util.Try
+
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.meta.io.AbsolutePath
+
+import com.google.gson.JsonPrimitive
+import org.eclipse.lsp4j.Location
+import org.eclipse.{lsp4j => l}
+
+class StacktraceAnalyzer(
+    workspace: AbsolutePath,
+    definitionProvider: DefinitionProvider,
+    gotoicon: String,
```

Comment:
Let's use `Icons` instead of string here.

Reply:
ok